nixos/services/varnish/default.nix

```diff
@@ -76,9 +76,18 @@ in {
     http_address = mkOption {
       type = types.str;
       default = "*:8008";
+      description = ''
```

Comment:
Adding a proper option description is good. We might want to document the approach of how to add multiple listen addresses here as well.

Reply:
https://github.com/flyingcircusio/fc-nixos/pull/1035/files#diff-8853157fe79ca26de07ebefb6906f31f01516e5d7df621eb158402a52aa2b220R126 

is this ok?

Reply:
What about the format `'<address>:<port> -a <address>:<port> -a <address>:<port>'`? IIRC this is what an earlier version of the PR even expected as a baseline, so I'm rather certain we have that format in some setups.

But if the parser supports both formats, I'm fine with only suggesting the better looking one here in the docs.

Reply:
Yeah that second form is allowed by varnish but is pretty much a hack in this context, let's not mention this here.